.circleci/config.yml

```
@@ -26,6 +26,9 @@ jobs:
       - run: pip install -r requirements.txt
       - run: black --check .
       - run: mypy algosdk
+      - run: |
+          python setup.py check -s
+          ! python setup.py check -s 2>&1 | grep --quiet --ignore-case "warning"
```

@algochoi When I locally configured the warning to appear, I see the check passing (return code = 0).

Here's a _suggestion_ that works in my local testing.  I agree with retaining the earlier `setup.py check` invocation present for the user to see output.
 
```suggestion
          python setup.py check -s 2>&1 | (! grep -qEi 'error|warning')
```

---

Thanks for the check, I will commit the suggestion - both variants seem to work on my machine.